pkg/utils/utils.go

```diff
@@ -302,6 +304,30 @@ func getNetdevMTU(pciAddr string) int {
 	return mtu
 }
 
+func getNetDevMac(ifaceName string) string {
+	glog.V(2).Infof("getNetDevMac(): get Mac for device %s", ifaceName)
+	macFilePath := filepath.Join(sysClassNet, ifaceName, "address")
+	data, err := ioutil.ReadFile(macFilePath)
+	if err != nil {
+		glog.Warningf("getNetDevMac(): fail to read Mac file %s", macFilePath)
+		return ""
+	}
+
+	return strings.TrimSpace(string(data))
+}
+
+func getNetDevLinkSpeed(ifaceName string) string {
+	glog.V(2).Infof("getNetDevLinkSpeed(): get LinkSpeed for device %s", ifaceName)
+	speedFilePath := filepath.Join(sysClassNet, ifaceName, "speed")
+	data, err := ioutil.ReadFile(speedFilePath)
+	if err != nil {
+		glog.Warningf("getNetDevLinkSpeed(): fail to read Link Speed file %s", speedFilePath)
+		return ""
+	}
+
+	return strings.TrimSpace(string(data))
```

Comment:
nit: if we could append speed unit to the string, for example, it returns 10000, and becomes 10000 Mb/s after appending the unit.

Reply:
added